### ISSUE TITLE
BMF broken 2.0.19.1-19.1-a

### ISSUE THREAD

**Issue by edgarcosta** [LMFDB/lmfdb#4576]
https://beta.lmfdb.org/ModularForm/GL2/ImaginaryQuadratic/2.0.19.1/19.1/a/

```
Exception on /ModularForm/GL2/ImaginaryQuadratic/2.0.19.1/19.1/a/ [GET]
Traceback (most recent call last):
  File "/home/sage/sage-9.2/local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/sage/sage-9.2/local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/sage/sage-9.2/local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/sage/sage-9.2/local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/sage/sage-9.2/local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/sage/sage-9.2/local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/lmfdb/lmfdb-git-dev/lmfdb/bianchi_modular_forms/bianchi_modular_form.py", line 567, in render_bmf_webpage
    data = WebBMF.by_label(label, max_eigs=numeigs)
  File "/home/lmfdb/lmfdb-git-dev/lmfdb/bianchi_modular_forms/web_BMF.py", line 68, in by_label
    return WebBMF(data, max_eigs)
  File "/home/lmfdb/lmfdb-git-dev/lmfdb/bianchi_modular_forms/web_BMF.py", line 57, in __init__
    self.make_form(max_eigs)
  File "/home/lmfdb/lmfdb-git-dev/lmfdb/bianchi_modular_forms/web_BMF.py", line 190, in make_form
    curve_bc_parts = [split_lmfdb_label(lab) for lab in curve_bc]
  File "/home/lmfdb/lmfdb-git-dev/lmfdb/bianchi_modular_forms/web_BMF.py", line 190, in <listcomp>
    curve_bc_parts = [split_lmfdb_label(lab) for lab in curve_bc]
  File "/home/lmfdb/lmfdb-git-dev/lmfdb/elliptic_curves/web_ec.py", line 33, in split_lmfdb_label
    return lmfdb_label_regex.match(lab).groups()
AttributeError: 'NoneType' object has no attribute 'groups'
[2021-05-21 11:05:24 UTC] 500 error on URL https://beta.lmfdb.org/ModularForm/GL2/ImaginaryQuadratic/2.0.19.1/19.1/a/ ()
```

**Comment by JohnCremona** [comment]
This will be corrected in the PR I am now making.

**Comment by JohnCremona** [comment]
This can be closed when #4577 is merged.
